src/main/java/com/internetitem/logback/elasticsearch/PropertySerializer.java

```diff
@@ -0,0 +1,55 @@
+package com.internetitem.logback.elasticsearch;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.internetitem.logback.elasticsearch.util.AbstractPropertyAndEncoder;
+
+import java.io.IOException;
+
+class PropertySerializer<T> {
+    synchronized void serializeProperty(JsonGenerator jsonGenerator, T event, AbstractPropertyAndEncoder<T> propertyAndEncoder) throws IOException {
```

Comment:
Also I don't think this needs to be `synchronized`

Reply:
I removed this unnecessary synchronization